src/shared/components/EmptyFallback/EmptyFallback.styles.ts

```diff
@@ -0,0 +1,38 @@
+import styled from '@emotion/styled'
+
+import { colors, media, sizes } from '@/shared/theme'
+
+import { EmptyFallbackSizes } from './EmptyFallback'
+
+import { Text } from '../Text'
+
+export const Container = styled.div<{ variant?: EmptyFallbackSizes }>`
+  margin: ${({ variant }) => (variant === 'large' ? sizes(20) : sizes(13.5))} auto 0;
```

Comment:
Where do these margins come from? The design seems to have a constant 40px vertical margin

Reply:
That's what I suggested, but ok I will change this
![Zrzut ekranu 2021-08-11 o 10 40 28](https://user-images.githubusercontent.com/13706246/129001326-d5560e60-71b0-4640-87e3-0f1f7e292fdf.png)